index.yaml

```diff
@@ -1,6 +1,25 @@
 apiVersion: v1
 entries:
   vsphere-cpi:
+  - apiVersion: v2
```

Comment:
Yifeng is this part consistent with the auto-generated result, do we still need a second MR?